packages/plugin-dev/src/browser/hosted-plugin-controller.ts

```diff
@@ -247,7 +247,7 @@ export class HostedPluginController implements FrontendApplicationContribution {
     }
 
     private getErrorMessage(error: Error): string {
-        return error.message.substring(error.message.indexOf(':') + 1);
+        return error?.message?.substring(error.message.indexOf(':') + 1);
```

Comment:
This function returns `string` but what you wrote seem to indicate that it could return `undefined`. Something looks off with this return line.

Reply:
@marechal-p Thanks, you are correct - my bad about returning undefined. I added this because of in certain cases error or error.message would be undefined which resulted in exception which would cause debug failure where you needed to reload browser to solve it. So the check is IMO essential - I'm just adding || '' to the end like so:
```
private getErrorMessage(error: Error): string {
        return error?.message?.substring(error.message.indexOf(':') + 1) || '';
    }
```